test_runner/README.md

```diff
@@ -11,6 +11,8 @@ Prerequisites:
       `pytest-3`.)
     - Install using something like this:
         - `pip3 install pytest psycopg2` (Debian or Ubuntu)
+    - Or, if you use `pipenv`, with
+	    - `pipenv install`
```

Comment:
I would better stick to pipenv only -- manual installation is very fragile in python. If we want support installation without pipenv better solution would be to have requirements.txt, which can be fed to pip directly

Reply:
Agree. I updated readme accordingly and merged.